src/seaport.ts

```diff
@@ -1244,23 +1254,21 @@ export class OpenSeaPort {
   /**
    * Compute the fees for an order
    * @param param0 __namedParameters
-   * @param assets Array of addresses and ids that will be in the order
+   * @param asset Addresses and id of asset (null if a bundle, unless all assets are from the same contract, then the first asset)
    * @param assetContract Optional prefetched asset contract (including fees) to use instead of assets
    * @param side The side of the order (buy or sell)
    * @param isPrivate Whether the order is private or not (known taker)
    * @param extraBountyBasisPoints The basis points to add for the bounty. Will throw if it exceeds the assets' contract's OpenSea fee.
    */
   public async computeFees(
-      { assets, assetContract, side, isPrivate = false, extraBountyBasisPoints = 0 }:
-      { assets?: Asset[];
+      { asset, assetContract, side, isPrivate = false, extraBountyBasisPoints = 0 }:
+      { asset: OpenSeaAsset | null;
```

Comment:
we should just use `OpenSeaAsset?` so that we don't need `asset: null` everywhere

Reply:
We sometimes pass in null in _makeBundleBuyOrder and _makeBundleSellOrder, if the bundle is not all from the same contract